# Request
---
# Response
int32 robot_count